tests/src/index.js

```diff
@@ -0,0 +1 @@
+console.log('Hello World');
```

Comment:
I would also add the syntax features that the build script should support. I think the following should be enough:

```js
console.log(
  'Hello',
  'World', // traling function comma
);

class A {
  foo = 'bar'; // class properties
}

async function shouldBeGenerator() {
  return null;
}
```

Ideally, the tests would not only test that the build script doesn't fail but also that the transpiled output is correct. Don't know how exactly now but we can do that in the next iteration!